kornia/augmentation/container/augment.py

```diff
@@ -7,7 +7,8 @@
 
 from kornia.augmentation.base import _AugmentationBase, GeometricAugmentationBase2D, IntensityAugmentationBase2D
 from kornia.constants import DataKey
-from kornia.geometry import transform_boxes, transform_points
+from kornia.geometry.bbox import transform_bbox
```

Comment:
Why not plural? We used transform_points as well.

Reply:
Discussed on slack.